src/components/BtnCtA.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./BtnCta.css";
+
+const Button = ({ variant, text }) => {
+  return (
+    <button className={`btn btn--cta btn--${variant}`} type="button">
+      {text}
```

Comment:
Another way you can display content inside a React component is to use `props.children`

We could do something like so: 

Define like this:
```js
const Button = (props) => {
  return (
    <button>{props.children}<button>
  )
}
```

Use like this: 

```
<Button>
  <SomeIconOnTheButton />
  Some text on the button
</Button>
```

This makes it easier to reuse the button easily